ext/vulkan/vulkan_core.h

```diff
@@ -4980,7 +4980,7 @@ typedef struct VkSparseImageMemoryRequirements2 {
 typedef struct VkPhysicalDeviceFeatures2 {
     VkStructureType             sType;
     void*                       pNext;
-    VkPhysicalDeviceFeatures    features;
+    VkPhysicalDeviceFeatures    standard;
```

Comment:
I don't mind this, but should we actually change this header file?  We could still call our struct standard either way.

-[Unknown]

Reply:
Oops, that was not intentional, heh. A global rename went haywire and I misse this damage.